CHANGELOG.md

```diff
@@ -12,12 +12,15 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Breaking changes
 
 ### New features & improvements
-- Make the test `status` not set by default ([#144](https://github.com/personio/datadog-synthetic-test-support/pull/144))
 
 ### Bug fixes
 
 ### Dependencies
 
+## [2.2.1] - 2023-10-23
```

Comment:
Should not this change rather be treated as a a breaking change and released as a new major version, since it might require users to modify their code if they wish to keep the old behaviour setting the status to "paused"?

Reply:
This change impacts only API test creation, browser tests are still creating via old approach. And it shouldn't impact the users, cause the default Datadog behavior is to keep the status unchanged. So if they have tests which are paused, this won't change anyways, they don't need to add `status()` function explicitly for it.

Reply:
Agreed to update the version to 2.3.0 since it impacts those who would like to keep the existing behavior.